src/game.cpp

```diff
@@ -5727,11 +5730,11 @@ int game::mon_info(WINDOW *w)
         _("North:"), _("NE:"), _("East:"), _("SE:"),
         _("South:"), _("SW:"), _("West:"), _("NW:")
     }};
-    std::array<int, 8> widths;
+    std::array<int, 8> widths{};
```

Comment:
I suggest backing these out, a number of compilers choke on it for questionable reasons.

Reply:
Sure thing.